apps/website/src/components/Playground/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Let's convert this to be TypeScript code. I pushed changes in config to this branch.

P.S. It's my mistake that I didn't do this in previous PR.